Automattic/jetpack
# Search: Update number formatting across Record Meter

Fixes #25004 

#### Changes proposed in this Pull Request:


Changes uses of `toLocaleString()` to `numberFormat()` across the Record Meter so all numbers are formatted using the site locale instead of the browser/system locale.

Before:

![NFBefore](https://user-images.githubusercontent.com/40267301/177945625-3d549f0c-4f0c-4484-9617-07b433c7d054.png)

After:

![NFAfter](https://user-images.githubusercontent.com/40267301/177945643-18ef60c0-6c21-4e6d-a7ea-bea725f65ada.png)


#### Other information:


#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Set up a test site with Jetpack installed.
2. Add a Search plan.
3. Change the site language to French.
4. Install the updated language files from `/wp-admin/update-core.php`. Only visible after changing the site language.
5. Visit the Record Meter at `/wp-admin/admin.php?page=jetpack-search&features=record-meter`.
6. Confirm the numbers are formatted to the site locale (in this case, French) instead of to the site locale.
